src/Symfony/Bundle/FrameworkBundle/Resources/config/web.xml

```diff
@@ -17,6 +17,29 @@
             <argument type="service" id="logger" on-invalid="ignore" />
         </service>
 
+        <service id="argument_resolver" class="Symfony\Component\HttpKernel\Controller\ArgumentResolver" public="false">
+            <argument type="service" id="argument_metadata_factory" />
+            <argument type="collection" />
+        </service>
+
+        <service id="argument_metadata_factory" class="Symfony\Component\HttpKernel\ControllerMetadata\ArgumentMetadataFactory" public="false" />
```

Comment:
I thought there were a kind of convention to write services used as arguments before being used as arguments. I was wrong ?